src/icalendar/parser.py

```diff
@@ -109,8 +109,8 @@ def param_value(value):
 # Could be improved
 
 # [\w-] because of the iCalendar RFC
-# \. because of the vCard RFC
-NAME = re.compile('[\w\.-]+')
+# . because of the vCard RFC
```

Comment:
This change is invalid... `.` matches any character in regex.

Reply:
The dot is inside square brackets, so it loses its special meaning. In general, I find fewer escape sequences easier to read. From [re docs](https://docs.python.org/3/library/re.html)

> Special characters lose their special meaning inside sets. For example, [(+*)] will match any of the literal characters '(', '+', '*', or ')'.

If you still prefer the backslash, let me know. I can add it back in.